src/state/_redux/user/reducer.ts

```diff
@@ -0,0 +1,17 @@
+import { createReducer } from 'typesafe-actions';
+import { User } from './types';
+import { loginUser } from './actions';
+
+type Action = any;
```

Comment:
Any.
Maybe something like
import PolicyModule from '@@state/Policy/module';

const initialState = new PolicyModule();

export default createReducer(initialState)

while module is just user types